docs/shared/CustomWarning.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+export default function CustomWarning({children}) {
```

Comment:
the reason we need a custom component is that the default admonition is hard to read in light mode:

![CleanShot 2021-06-02 at 18 00 22](https://user-images.githubusercontent.com/6764957/120461270-7181aa80-c3cc-11eb-94ae-a30f4096e3b3.png)

here it is with the custom component

![CleanShot 2021-06-02 at 18 00 47](https://user-images.githubusercontent.com/6764957/120461322-7c3c3f80-c3cc-11eb-91d0-92e73691828f.png)


this is a known issue for docusaurus but they are still working on it https://github.com/facebookincubator/infima/issues/55